### PR TITLE
Integrate Alexa Skill Events

### PR DESCRIPTION
**Pull request by wuelcas**
Integrate request handlers for [Alexa Skill Events](https://developer.amazon.com/public/solutions/alexa/alexa-skills-kit/docs/skill-events-in-alexa-skills#skill-events-in-json-format)

### CODE REVIEW

**Comment by omenocal** [comment]
Oh something more, update the documentation !

File: docs/statemachine-skill.rst [hunk 1]
```diff
@@ -243,3 +243,68 @@ Handle events from the `AudioPlayer interface <https://developer.amazon.com/publ
...
+  :param object reply: A reply to be sent as the response
+  :returns object reply: Alexa only needs an acknowledgement to you received and processed the event so it doesn't need to resend the event. Just returning the :ref:`reply <reply>` object is enough
+
+  This is an example on how your skill can process a :js:func:`~Voxa.onAlexaHouseholdListEvent.ItemsCreated` event.
+
+  .. code-block:: javascript
+
+    skill['onAlexaHouseholdListEvent.ItemsCreated']((alexaEvent, reply) => {
+      const listId = alexaEvent.request.body.listId;
+      const userId = alexaEvent.context.System.user.userId;
```
Comment (armonge) [inline docs/statemachine-skill.rst]: always use [`alexaEvent.user.userId`](http://voxa.readthedocs.io/en/stable/alexa-event.html#AlexaEvent.user)

Comment (wuelcas) [reply docs/statemachine-skill.rst]: I'll change it right now

**Comment by armonge** [comment]
@wuelcas why did you close this?

**Comment by wuelcas** [comment]
I was talking with @omenocal and checked why Travis was failing, it was because of an incompatible module but it doesn't say what module is it. We think this happened because I made the changes over master instead of the dev branch and when I tried to update the master branch with the dev branch, a commit made by you appear in the PR. I will open another pull request but I'll make the changes in the dev branch to avoid any conflicts

**Pull request closed without merging**